src/components/navigation/Navbar.tsx

```diff
@@ -1,12 +1,25 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import "./Navbar.scss";
 
 export const Navbar: React.FunctionComponent = () => {
   return <nav>
-    <ul>
-      <li><Link to="/">Home</Link></li>
-      <li><Link to="/rovers">Rovers</Link></li>
-      <li><Link to="/quiz">Quiz</Link></li>
-    </ul>
+
+    <div className="emptySpacingBox"></div>
+
+    <div id="menuToggle">
+      <input type="checkbox" />
+      
+        <span></span>
+        <span></span>
+        <span></span>
```

Comment:
So... this is a very common way of doing this, which is why I'm not going to ask you not to do it, but hopefully you can see that having empty HTML elements purely for visuals isn't very semantic/accessible! Don't change this at this point, but I think the most accessible way of doing this is to use an SVG icon, possibly animating it somehow.

(You're probably noticing at this point that I value accessibility over most other things! 😉)

Reply:
Noted